tests/cupy_tests/lib_tests/test_polynomial.py

```diff
@@ -231,6 +235,123 @@ def test_poly1d_div_scalar(self, xp, dtype):
         b = xp.poly1d(a) / 10
         return b.coeffs
 
+    @testing.for_all_dtypes()
+    @testing.numpy_cupy_equal()
+    def test_poly1d_add_numpy_scalar_numpy_scalar(self, xp, dtype):
+        with cupyx.allow_synchronize(False):
+            return dtype(5) + dtype(10)
+
+    @testing.for_all_dtypes()
+    @testing.numpy_cupy_equal()
+    def test_poly1d_add_numpy_scalar_python_scalar(self, xp, dtype):
+        with cupyx.allow_synchronize(False):
+            return dtype(5) + dtype(10).item()
+
+    @testing.for_all_dtypes()
+    @testing.numpy_cupy_array_equal()
+    def test_poly1d_add_numpy_scalar_array(self, xp, dtype):
+        a = testing.shaped_arange((5,), xp, dtype)
+        with cupyx.allow_synchronize(False):
+            return dtype(5) + a
+
+    @testing.for_all_dtypes(no_bool=True)
```

Comment:
Why `no_bool` is needed?
```suggestion
    @testing.for_all_dtypes(no_bool=True)
```

Reply:
The test fails for numpy.bool_